app/controllers/posts_controller.rb

```diff
@@ -35,6 +35,19 @@ def create
     end
```

Comment:
Great job @Josphat205 🤝

- [ ] As stated in the project requirements, kindly use `CanCanCan` for Post authorization in Post Controller. You may add something like this line of code in your posts controller `load_and_authorize_resource`.